src/Languages/Dockerfile/DockerfileLanguage.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Highlight\Languages\Dockerfile;
+
+use Tempest\Highlight\Languages\Base\BaseLanguage;
+use Tempest\Highlight\Languages\Dockerfile\Patterns\ImageAliasKeywordPattern;
+use Tempest\Highlight\Languages\Dockerfile\Patterns\ImageAliasNamePattern;
+use Tempest\Highlight\Languages\Dockerfile\Patterns\ImageNamePattern;
+use Tempest\Highlight\Languages\Dockerfile\Patterns\ImageTagPattern;
+use Tempest\Highlight\Languages\Dockerfile\Patterns\KeywordPattern;
+
+class DockerfileLanguage extends BaseLanguage
+{
+    public function getName(): string
+    {
+        return 'Dockerfile';
```

Comment:
Should this be `dockerfile`? Maybe add `docker` as an alias as well?

Reply:
I had the same thought as you regarding the alias. And ChatGPT would also describe it this way, so I think `dockerfile` and `docker` will be enough.